sdk/PowerBI.AspNet.Mvc/Html/ReportExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static class ReportExtensions
         /// <param name="report">The Power BI report</param>
         /// <param name="htmlAttributes">Additional attributes used while rendering the report</param>
         /// <returns></returns>
-        public static MvcHtmlString PowerBIReport(this HtmlHelper htmlHelper, string name, Report report, object htmlAttributes = null)
+        public static IHtmlString PowerBIReport(this HtmlHelper htmlHelper, string name, Report report, object htmlAttributes = null)
```

Comment:
As mentioned in #34, before making the change, we should discuss that it's potentially breaking for any consumers who might be doing the following:

`MvcHtmlString foo = htmlHelper.PowerBIReport("myReport", this.viewModel.Report);`

This seems highly unlikely as well as trivial to resolve.